src/index.ts

```diff
@@ -123,6 +123,15 @@ export class GraphQLServer {
 
     this.options = { ...this.options, ...options }
 
+    if (this.options.subscriptions) {
+      if (typeof this.options.subscriptions === 'string') {
+        this.options.subscriptions = { path: this.options.subscriptions }
```

Comment:
Would be great if you could implement it in a way without overwriting/changing the `this.options` object since changing this in the future might be a breaking change.